assets/js/atomic/blocks/product/save.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+const Save = ( { attributes } ) => {
+	return (
+		<div className={ classnames( 'is-loading', attributes.className ) } />
```

Comment:
This is new; saves some markup for the InnerBlock so we can render from this on the frontend instead of using layoutconfig.